src/eredis_client.erl

```diff
@@ -486,13 +491,13 @@ reconnect_loop(Client, #state{reconnect_sleep = ReconnectSleep} = State) ->
             [Client ! M || M <- Msgs];
         {error, _Reason} ->
             timer:sleep(ReconnectSleep),
-            reconnect_loop(Client, State);
+            reconnect_loop(Client, State, Options);
```

Comment:
export this function, and loop over with `?MODULE:reconnect_loop(Client, State, Options)`